numpy/add_newdocs.py

```diff
@@ -4289,16 +4295,17 @@ def luf(lamdaexpr, *args, **kwargs):
     """
     a.setflags(write=None, align=None, uic=None)
 
-    Set array flags WRITEABLE, ALIGNED, and UPDATEIFCOPY, respectively.
+    Set array flags WRITEABLE, ALIGNED, (WRITEBACKIFCOPY and UPDATEIFCOPY),
+    respectively.
 
     These Boolean-valued flags affect how numpy interprets the memory
     area used by `a` (see Notes below). The ALIGNED flag can only
     be set to True if the data is actually aligned according to the type.
-    The UPDATEIFCOPY flag can never be set to True. The flag WRITEABLE
-    can only be set to True if the array owns its own memory, or the
-    ultimate owner of the memory exposes a writeable buffer interface,
-    or is a string. (The exception for string is made so that unpickling
-    can be done without copying memory.)
+    The (deprecated) UPDATEIFCOPY and WRITEBACKIFCOPY flags can never be set
```

Comment:
"The WRITEBACKIFCOPY and (deprecated) UPDATEIFCOPY", to make clear that we're not saying both are deprecated.